recipes/llvm-core/all/patches/11x-native.patch

```diff
@@ -0,0 +1,81 @@
+--- CMakeLists.txt
++++ CMakeLists.txt
+@@ -943,10 +943,6 @@
+ include_directories( ${LLVM_INCLUDE_DIR} ${LLVM_MAIN_INCLUDE_DIR})
+ 
+ # when crosscompiling import the executable targets from a file
+-if(LLVM_USE_HOST_TOOLS)
+-  include(CrossCompile)
+-  llvm_create_cross_target(LLVM NATIVE "" Release)
```

Comment:
This was a fun one. The reason for some of the failures is that **llvm** configures a second **cmake** project in parallel to compile some of the tools for the host architecture in case you're doing cross compiling (or simply when it does not detect any local **llvm-config** tool). That second parallel build is not configured so well, hence all of the missing stuff errors.

We don't need that tools (let's leave the work to **conan** 🦾), but turns out that there is no [way to disable them](https://reviews.llvm.org/D41580) through **cmake** configuration options. So I had no other choice but to patch everything out of the sources (it seems [I'm not alone](http://git.yoctoproject.org/cgit/cgit.cgi/meta-amd/tree/common/recipes-core/llvm/files/0004-Dont-build-llvm-config-and-tblgen-concurrently.patch?h=morty) on that).